environment/admin_elasticache.tf

```diff
@@ -1,33 +1,16 @@
 resource "aws_elasticache_replication_group" "admin" {
-  // Include this once first two subtasks of DDBP-3192 are done
-  //  count                         = local.account.is_production == 1 ? 1 : 0
-  automatic_failover_enabled    = true
+  automatic_failover_enabled    = local.account.elasticache_count == 1 ? false : true
   engine                        = "redis"
   engine_version                = "5.0.0"
-  availability_zones            = ["eu-west-1a", "eu-west-1b"]
+  availability_zones            = null_resource.elasticache_az_list.*.triggers.az
```

Comment:
I think we can just leave this blank and AWS will identify # suitable AZs in eu-west1 ([search "PreferredCacheClusterAZs" here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticache-replicationgroup.html)). Doesn't make a difference to what's actually built, but would save us a bit of configuration.